setup.cfg

```diff
@@ -1,2 +1,68 @@
-[flake8]
-ignore=E203,F403,E128,E126,E123,E121,E265,E501,N802,N803,N806,C901,D100,D102,D102,D10,W503,E731,E402,E741
+[metadata]
+name = Lektor
+description = A static content management system.
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = BSD
+platforms = any
+author = Armin Ronacher
+author_email = armin.ronacher@active-4.com
+url = http://github.com/lektor/lektor/
+classifiers =
+    Framework :: Lektor
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Internet :: WWW/HTTP :: Dynamic Content
+    Topic :: Software Development :: Libraries :: Python Modules
+requires = setuptools_scm
```

Comment:
is this valid?

Reply:
No.  I think it's cruft.  I'm not sure why I put it there.  Will delete.

Reply:
Deleted.    I'm not sure how that got there.  Thank you for pointing it out!